CSharp/Library/Microsoft.Bot.Builder/Resource/Resources.fr.resx

```diff
@@ -1,251 +1,251 @@
-    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
-  </resheader>
-  <data name="CommandBack" xml:space="preserve">
-    <value>Retour</value>
-  </data>
-  <data name="CommandBackHelp" xml:space="preserve">
-    <value>Retourner à la question précédente.</value>
-  </data>
-  <data name="CommandBackTerms" xml:space="preserve">
-    <value>retour; retourner</value>
-  </data>
-  <data name="CommandHelp" xml:space="preserve">
-    <value>Assistance</value>
-  </data>
-  <data name="CommandHelpHelp" xml:space="preserve">
-    <value>Assistance: Montrez les réponses possibles.</value>
-  </data>
-  <data name="CommandHelpTerms" xml:space="preserve">
-    <value>aide;choix; assistance; \?</value>
-  </data>
-  <data name="CommandQuit" xml:space="preserve">
-    <value>Abandonner</value>
-  </data>
-  <data name="CommandQuitHelp" xml:space="preserve">
-    <value>Abandonner: Abandonner sans finir</value>
-  </data>
-  <data name="CommandQuitTerms" xml:space="preserve">
-    <value>arrêter;finir;au revoir;salut;ciao;adios;abandonner</value>
-  </data>
-  <data name="CommandReset" xml:space="preserve">
-    <value>Recommencer</value>
-  </data>
-  <data name="CommandResetHelp" xml:space="preserve">
-    <value>Recommencer remplir le formulaire.  (Vos réponses précédentes sont enregistrées.)</value>
-  </data>
-  <data name="CommandResetTerms" xml:space="preserve">
-    <value>recommencer;réinitialiser;</value>
-  </data>
-  <data name="CommandStatus" xml:space="preserve">
-    <value>Statut</value>
-  </data>
-  <data name="CommandStatusHelp" xml:space="preserve">
-    <value>Statut: Montrer le progrès en remplissant le formulaire jusqu'à présent.</value>
-  </data>
-  <data name="CommandStatusTerms" xml:space="preserve">
-    <value>statut;progrès;les résultats jusqu'ici;statut actuel</value>
-  </data>
-  <data name="ConfirmChoiceFormat" xml:space="preserve">
-    <value>{1}</value>
-    <comment>0-number, 1-description</comment>
-  </data>
-  <data name="DefaultChoiceFormat" xml:space="preserve">
-    <value>{0}. {1}</value>
-    <comment>0-number, 1-description</comment>
-  </data>
-  <data name="DefaultChoiceLastSeparator" xml:space="preserve">
-    <value>, ou </value>
-  </data>
-  <data name="DefaultChoiceSeparator" xml:space="preserve">
-    <value>, </value>
-  </data>
-  <data name="DefaultLastSeparator" xml:space="preserve">
-    <value>, et </value>
-  </data>
-  <data name="DefaultSeparator" xml:space="preserve">
-    <value>, </value>
-  </data>
-  <data name="LanguageArticles" xml:space="preserve">
-    <value>un, une, la</value>
-  </data>
-  <data name="LanguageStopWords" xml:space="preserve">
-    <value></value>
-  </data>
-  <data name="MatchCurrentChoice" xml:space="preserve">
-    <value>Le choix actuel ("c"); courant; c; choix actuel</value>
-  </data>
-  <data name="MatchNo" xml:space="preserve">
-    <value>Non;pas;rien;2</value>

```

Comment:
>Non [](start = 11, length = 3)

these files are generated from xlf files. Make changes in xlf files and use Microsoft multilingual app toolkit to generate resources files.